WordPress/Classes/ViewRelated/Post/PostCompactCell.swift

```diff
@@ -204,3 +204,7 @@ extension PostCompactCell: GhostableView {
         static let opacity: Float = 0.5
     }
 }
+
+extension PostCompactCell: NibReusable {
+    static var defaultReuseID = "PostCompactCellIdentifier"
```

Comment:
FYI NibReusable has a default implementation for defaultReuseID so you don't need to provide an implementation :)

Reply:
Fixed!